tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -472,7 +472,7 @@ qos/test_qos_sai.py::TestQosSai:
   skip:
     reason: "Unsupported testbed type."
     conditions:
-      - "topo_name not in ['t0', 't0-64', 't0-116', 't0-35', 'dualtor-56', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-backend'] or asic_type not in ['mellanox']"
+      - "topo_name not in ['ptf32', 'ptf64', 't0', 't0-64', 't0-116', 't0-35', 'dualtor-56', 'dualtor', 't0-80', 't0-backend', 't1-lag', 't1-64-lag', 't1-backend'] or asic_type not in ['mellanox']"
```

Comment:
t0-56 is not supported? if it supports any topology, shouldn't we leave only the asic type validation?

Reply:
> t0-56 is not supported? if it supports any topology, shouldn't we leave only the asic type validation?

Hi, @roysr-nv  I only sure that the 'ptf32' and 'ptf64'' are supported, not sure if the t0-56 supported or not, so this is why I only add the ptf32 and ptf64 in the supported topo